src/packages/styles.css

```diff
@@ -203,7 +213,7 @@
   background: rgba(255, 255, 255, 0.3);
 }
 
-[data-sonner-toast] [data-close-button] {
+:where([data-sonner-toast]) :where([data-close-button]) {
```

Comment:
I intentionally modified this part to not keep it consistent with `sonner`, using `:where` as the selector. This is because it fails in dark mode.

We may need to find a way to ensure the close button displays correctly in dark mode.

![CleanShot 2024-11-14 at 10 47 52@2x](https://github.com/user-attachments/assets/e055c316-5f8c-4c07-944c-b9b93d267a0f)